src/plugins/vis_augmenter/README.md

```diff
@@ -0,0 +1 @@
+Contains interfaces, type definitions, helper functions, and services used for allowing external plugins to augment Visualizations. Registers the relevant saved object types and expression functions.
```

Comment:
This could definitely use some more documentation (and interactive examples) to demonstrate how plugins can connect to and use it.

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3383